web/docs/database/backends.md

```diff
@@ -1,3 +1,182 @@
 ---
-title: Database Backends
+title: Databases
 ---
+
+[Entities](/docs/database/entities.md), [Operations](/docs/database/operations.md) and [Automatic CRUD](/docs/database/crud.md) together make a high-level interface for working with your app's data. Still, all that data has to live somewhere, so let's see how Wasp deals with databases.
+
+## Database configuration
+
+You can tall Wasp which database to use in the `app` declaration's `db` field:
+
+```wasp
```

Comment:
I usually added Javascript/Typescript tabs for Wasp files as well to keep ti consistent. Also, the user can toggle and see, oh okay, I should put the `.js` extension even when I'm using Typescript.